src/grpc/iot_config/hpr_route_ets.erl

```diff
@@ -382,9 +380,9 @@ delete_skf(SKF) ->
     ok.
 
 -spec lookup_skf(ETS :: ets:table(), DevAddr :: non_neg_integer()) ->
-    [{SessionKey :: binary(), Timestamp :: integer(), MaxCopies :: non_neg_integer()}].
+    [{SessionKey :: binary(), MaxCopies :: non_neg_integer()}].
 lookup_skf(ETS, DevAddr) ->
-    MS = [{{{'$1', '$2'}, {DevAddr, '$3'}}, [], [{{'$2', '$1', '$3'}}]}],
+    MS = [{{'$2', {DevAddr, '$3'}}, [], [{{'$2', '$3'}}]}],
```

Comment:
do these matchspec placeholders need to reset now that you're not returning three values?

Reply:
They only need to match. But it's better to start them from 1.